jenkins/issuer-kit-admin/Jenkinsfile

```diff
@@ -1,4 +1,4 @@
-def PIPELINE_NAME = "issuer-web"
+def PIPELINE_NAME = "issuer-kit-admin"
 
 def LIB_NAME= "ised-business-banking-lib"
```

Comment:
`LIB_NAME` should be changed to `identity-kit` - also in other files.

Reply:
updated LIB_NAME and cleaned up 4a9599-tools